apps/i18n/common/en_us.json

```diff
@@ -1670,7 +1670,7 @@
   "loginInfo_joinStep1": "Create a Code.org account if they haven’t already done so. They can do this at {url}. Note that they can either sign up with an email address and password, or sign up through Google, Facebook, or Microsoft by clicking on one of these buttons:",
   "loginInfo_joinStep1Buttons": "Screenshot of three buttons, reading 'Continue with Google', 'Continue with Facebook', and 'Continue with Microsoft'",
   "loginInfo_joinStep2": "Sign in to their Code.org account.",
-  "loginInfo_joinStep3": "Navigate to {url} and type in their section code: {code}.",
+  "loginInfo_joinStep3": "Navigate to {url}/{code}.",
```

Comment:
I wonder if we should just make it a new key with solely the `url` tag to potentially avoid translators messing up the proper order.

Reply:
Good point. It could also potentially disrupt the context of the string in some languages

Reply:
<img src="https://github.com/user-attachments/assets/ef40d6ee-df1f-4db7-b62f-11b9ba2712af">

<img src="https://github.com/user-attachments/assets/42c2efc0-f451-40e6-807e-7d32f7eb32df">

Reply:
Good call; I changed it to just be a single param in the string.